simulation/robot_calibration.py

```diff
@@ -30,6 +30,12 @@
 win.setWindowTitle("PID loop monitor")
 win.resize(1080, 720)
 
+def deg2rad(deg):
+    return deg * 2. * math.pi / 360.
+
+def rad2deg(rad):
+    return rad * 360. / (2 * math.pi)
+
```

Comment:
As you import math with not using builtin functions ?
https://docs.python.org/2/library/math.html#angular-conversion

Reply:
Ok, will update thx